tests/e2e-k8s/spikes/sp799-ingress/run.sh

```
@@ -0,0 +1,189 @@
+#   (B) iroh-direct — no LB at all; external subscriber dials via iroh relay/NAT
+#       traversal from inside the pod's netns out
+#
+# STATUS: harness/plumbing validated on the dev box; the cert-pinned QUIC layer
+# needs a QUIC stand-in image + multi-node (gated on the inotify sysctl fix,
+# see ../../FINDINGS.md). This script is the runnable design: once STANDIN_IMAGE
+# is built and the cluster is multi-node, set the env vars and run end-to-end.
+set -euo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+# shellcheck source=../../lib/common.sh
+source "${SCRIPT_DIR}/../../lib/common.sh"
```

Shellcheck (0.11.0)

[info] Not following: ../../lib/common.sh was not specified as input (see shellcheck -x).

(SC1091)